Classes/common/CDTIncrementalStore.m

```diff
@@ -2208,6 +2221,75 @@ - (id)executeSaveRequest:(NSSaveChangesRequest *)saveRequest
     return @[];
 }
 
+- (id)executeBatchUpdateRequest:(NSBatchUpdateRequest *)updateRequest
```

Comment:
Now that the `CDTISBatchUpdateResult` is dead, can we return the right type, or at least the promised comment :smile: